Triphouse-project/index.html

```diff
@@ -17,7 +17,7 @@
                     <img src="./ikons/logo.svg" alt="logo" />
```

Comment:
![2021-04-26_21h04_02](https://user-images.githubusercontent.com/19935417/116130546-07425f80-a6d4-11eb-863e-9096ec383650.png)
![2021-04-26_21h01_43](https://user-images.githubusercontent.com/19935417/116130578-11fcf480-a6d4-11eb-9f47-e27c2b68c3f0.png)


Check paddings, width, and other properties for font-size.

Reply:
done